amplify/backend/function/getMembersForColony/src/reputation.Js

```diff
@@ -0,0 +1,35 @@
+const { BigNumber } = require('ethers');
+
+const ZERO = '0';
+const NEARZERO = '~0';
+const DECIMAL_PLACES = 2;
+
+const calculatePercentageReputation = (
```

Comment:
Why is this file's extension starting with a capital `J` ?

![Screenshot from 2022-12-01 15-05-03](https://user-images.githubusercontent.com/1193222/205060191-fc87117d-8c96-47fc-b631-fe08e5a8e855.png)